packages/gatsby/src/utils/jobs-manager.ts

```diff
@@ -192,17 +211,19 @@ const runJob = (job, forceLocal = false) => {
   }
 }
 
+function isInternalJob(job: JobInput | InternalJob): job is InternalJob {
+  return (job as InternalJob).id !== undefined
```

Comment:
The original code calso checked for `contentDigest`, was that intentionally dropped?

Reply:
Good spot, I think I missed that. I can't think of a good reason why that would've been removed.